### ISSUE TITLE
Too many build_tree()

### ISSUE THREAD

**Issue by bensadeghi** [bensadeghi/DecisionTree.jl#65]
The original ```DecisionTree.build_tree``` functions are now just wrappers for the new ```treeclassifier.build_tree``` and  ```treeregressor.build_tree``` routines.
This naming convention makes it unnecessarily confusing and hard to follow.

May I suggest renaming the new functions to  ```treeclassifier.build``` and  ```treeregressor.build```, or similiar ?

cc @Eight1911

**Comment by Eight1911** [comment]
How about `treeclassifier.fit`? `treeclassifier.build` is also good.

**Comment by bensadeghi** [comment]
@Eight1911 - Excellent! I'll change them to ```fit```
